src/main/java/Duke.java

```diff
@@ -1,10 +1,333 @@
+    }
+
+
+    /*
+     * ===========================================
+     *           MAIN COMMANDS
+     * ===========================================
+     */
+
+
+    /**
+     * Execute commands as intended by user
+     *
+     * @param currentTasks Tasks Array of all tasks
+     * @param numOfTasks   Number of tasks
+     * @param isOngoing    Whether the program has yet to be exited
+     * @return numOfTasks Updated number of tasks
+     * @throws DukeException to catch error
+     */
+    private static void executeCommands(Task[] currentTasks, int numOfTasks, boolean isOngoing) throws DukeException {
+        String userInput;
+        while (isOngoing) {
+            userInput = getUserInput();
+            String[] splitUserInput = splitCommands(userInput);
+            switch (splitUserInput[0].toLowerCase()) {
+            case "bye":
+                printExitMessage();
+                isOngoing = false;
+                break;
+            case "list":
+                commandList(currentTasks, numOfTasks);
+                break;
+            case "done":
+                markAsDone(currentTasks, numOfTasks, splitUserInput[1]);
+                break;
+            case "deadline":
+                numOfTasks = commandDeadline(currentTasks, numOfTasks, splitUserInput[1]);
+                break;
+            case "todo":
+                numOfTasks = commandToDo(currentTasks, numOfTasks, splitUserInput[1]);
+                break;
+            case "event":
+                numOfTasks = commandEvent(currentTasks, numOfTasks, splitUserInput[1]);
+                break;
+            case "help":
+                commandHelp();
+                break;
+            default:
+                try {
+                    throw new DukeException("UNINTELLIGIBLE_COMMAND");
+                } catch (DukeException exception) {
+                    printSeparator();
+                    System.out.println(exception);
+                    printSeparator();
+                }
+            }
+        }
+    }
+
+    /**
+     * Prints all tasks in task list
+     *
+     * @param currentTasks Tasks Array of all tasks
+     */
+    private static void commandList(Task[] currentTasks, int numOfTasks) {
+        printSeparator();
+        if (numOfTasks == 0) {
+            System.out.println("You have zero task at hand!");
+        }
+        for (int i = 0; i < numOfTasks; i++) {
+            System.out.println((i + 1) + "." + addBrackets(currentTasks[i].getType()) +
+                    addBrackets(currentTasks[i].getStatusIcon()) + " " + currentTasks[i].getDescription());
+        }
+        printSeparator();
+    }
+
+    /**
+     * Adds a Tasks.Deadline task to the currentTasks Tasks.Task array
+     *
+     * @param currentTasks Tasks Array of all tasks
+     * @param numOfTasks   Number of tasks
+     * @param userInput    User input
+     * @return numOfTasks Updated number of tasks
+     */
```

Comment:
Nice documentation! :) I can immediately know what the method is trying to do just by reading this instead